src/proto/error.rs

```diff
@@ -38,6 +38,8 @@ pub enum ZkError {
     NotReadOnly = -119,
     /// Attempt to remove a non-existing watcher.
     NoWatcher = -121,
+    /// No error occurred.
+    Ok = 0,
```

Comment:
I'm surprised this has to be registered here as an error? From memory, the implementation specifically checks for `0` as a success, no?

Reply:
Yep, your memory is totally correct—the reason I added this is because ZK multi requests use this state to indicate "this request was perfectly fine, but it didn't cause any changes because the multi request as a whole didn't commit." (See: https://github.com/jonhoo/tokio-zookeeper/pull/15/files#diff-baa08a2103c5b392637fc0ae1fcab2abR138)

Reply:
I've changed the place that used to check for `errcode != 0` to check for `errcode != ZkError::Ok` for the sake of consistency. Open to other ideas if you've got something cleaner, though!

Reply:
No, that seems fine!